parser/src/test/resources/schemas/DefaultValues.avsc

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "record",
+  "name": "AdStatisticsEvent",
+  "namespace": "com.linkedin.events.pinot.derived.ads",
+  "doc": "Events emitted for Ad campaign performance metrics and consumed by Pinot. Very similar to AdAnalyticsEvent, but with no demographic data.",
+  "fields": [
+    {
+      "name": "approximateUniqueImpressions",
+      "type": "bytes",
+      "default": "\u0000\u0000\u0000\u0008\u0000\u0000\u0000\u00AC\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000",
+      "doc": "An approximate count of unique members that received impressions from from the ad unit. This field is a byte representation of a HyperLogLog (HLL). For more on HLLs refer to https://docs.google.com/document/d/1CUE6F8EyOC4pP3uguIXedb3POWhVST7BL0d6rZ1HOAc/.  The default value is the byte representation of an empty HyperLogLog",
+      "compliance": "NONE"
+    },
+    {
+      "name": "approximateUniqueLandingPageClicks",
```

Comment:
can we clean up these tests to use generic avro and not linkedin-specific avro & doc comments?

Reply:
agree, create simple schemas for testing. Do not use real Linkedin schemas.